### DISCUSSION TITLE
Expo Deployment gives 404: NOT_FOUND

### DISCUSSION THREAD

**Discussion by rhutikcodes** [vercel/community#7260] [Help]
### Summary

My Expo project is successfully built but when i visit the app, it says 404: NOT_FOUND

<img width="1024" alt="Screenshot 2024-06-22 at 11 41 49 PM" src="https://github.com/vercel/community/assets/46106467/ac131c91-d44e-4924-ae09-b5906cd110ab">

### Example

https://nxtjob-community.vercel.app/

### Steps to Reproduce

Vercel.json file

`{
    "buildCommand": "npx tailwindcss -i ./src/global.css -o ./node_modules/.cache/nativewind/global.css.web.css && expo export -p web",
    "outputDirectory": "dist",
    "devCommand": "expo start --web",
    "installCommand": "npm install --legacy-peer-deps",
    "cleanUrls": true,
    "framework": null,
    "rewrites": [
        {
            "source": "/:path*",
            "destination": "/"
        }
    ]
}
`

**Comment by danger9224** [comment]
Have you found a solution to this problem yet? For additional context, I've configured the `server` option inside `app.json`:

```
"web": {
  "bundler": "metro",
  "output": "server",
},
```

[a comment by an automated account is not shown]
